utils/get-grpc.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
I reeeeally have a hate for these fetch/install scripts, and would have a strong preference towards doing it in any other way, if possible.

Did you look into a fetchcontent-based approach? Looks like it's supported: https://github.com/grpc/grpc/tree/b217753062370b8f12e72fc3414eabe0a6f9f16e/src/cpp#fetchcontent

Reply:
Yeah. Tried that and it failed spectacularly. gRPC pulls in like 16 submodules for 16 different dependencies. Some of them use the basel build system, so that gets built as well... One of the dependencies failed to build and I just knew that after I fix that, another would fail and didn't want to go down that rabbit hole.

I'm hoping that most distros will have a recent enough version of gRPC that we can just use the system-installed one.

Reply:
Yikes, that sounds horrible... then this does seem like the lesser evil. Thank you for also investigating that!

Reply:
I'll add that if someone wants to get this working in the future (as an alternative to a separate install), I have no major objections. (Just philosophical ones related to the very idea of FetchContent.)